### PR TITLE
Update the CHANGES.md

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+* Port to Cesium 1.7.1 and Ol 3.3.0
+
+# Breaking changes
+  * Require the `olcs.OLCesiium.setEnabled` parameter to be defined.
```
Comment (ahocevar) [inline CHANGES.md]: Should be "The `enable` argument to `olcs.OLCesium#setEnabled()` is no longer optional."

**Comment by ahocevar** [comment]
@gberaudo Thanks for this fix. You have a typo in there (Cesiium). While at it, you may want to change the wording a bit, e.g. to what I suggested in the comment.

**Comment by gberaudo** [comment]
Thank you for the review @ahocevar; I used your nice wording.

**Comment by ahocevar** [comment]
Very nice, please merge!

**Pull request merged**